### ISSUE TITLE
Allow `set` to write the result back to a file.

### ISSUE THREAD

**Issue by skarekrow** [allanjude/uclcmd#8]
This can either be default behavior or an optional switch if it's deemed experimental. 

Currently the process is:
```
uclcmd set -f /path/to/something foo bar > /path/to/something.tmp ; mv /path/to/something.tmp /path/to/something.
```
The process would be more robust in a script with error code checking and such, but that's the procedure currently. It does not work for non-existing keys.

So ideally the process would be:
```
uclcmd set [-w] -f /path/to/something foo bar
```
With `bar` replacing any previous value of `foo`, or if `foo` does not exist, adding that key value pair.

**Comment by allanjude** [comment]
Fixed in #12

**Comment by skarekrow** [comment]
Thanks! :+1:
